manifests/2.15.0/opensearch-dashboards-2.15.0-test.yml

```diff
@@ -5,7 +5,111 @@ ci:
   image:
     name: opensearchstaging/ci-runner:ci-runner-rockylinux8-opensearch-dashboards-integtest-v4
 components:
-  - name: OpenSearch-Dashboards
+  - name: OpenSearch-Dashboards-ci-group-1
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+      additional-cluster-configs:
+        vis_builder.enabled: true
+        data_source.enabled: true
+        savedObjects.maxImportPayloadBytes: 10485760
+        server.maxPayloadBytes: 1759977
+        logging.json: false
+        data.search.aggs.shardDelay.enabled: true
+        csp.warnLegacyBrowsers: false
+  - name: OpenSearch-Dashboards-ci-group-2
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+      additional-cluster-configs:
+        vis_builder.enabled: true
+        data_source.enabled: true
+        savedObjects.maxImportPayloadBytes: 10485760
+        server.maxPayloadBytes: 1759977
+        logging.json: false
+        data.search.aggs.shardDelay.enabled: true
+        csp.warnLegacyBrowsers: false
+  - name: OpenSearch-Dashboards-ci-group-3
```

Comment:
Hey @SuZhou-Joe may I know what tests or components part of these groups ?  example for `OpenSearch-Dashboards-ci-group-3`, coming from this PR https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1411/files, how to know what tests or components are part of `OpenSearch-Dashboards-ci-group-3`?

Reply:
The specs are written in the `package.json`. 
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/package.json#L28-L36

Reply:
I would like to get this change documentation somewhere (may be in @peterzhuamazon should we update the https://github.com/opensearch-project/opensearch-build/wiki/Testing-the-Distribution with this change, adding few lines on how does this grouping work).
Thanks